x-pack/platform/plugins/shared/actions/server/config.ts

```diff
@@ -121,9 +121,22 @@ export const configSchema = schema.object({
   microsoftGraphApiScope: schema.string({ defaultValue: DEFAULT_MICROSOFT_GRAPH_API_SCOPE }),
   microsoftExchangeUrl: schema.string({ defaultValue: DEFAULT_MICROSOFT_EXCHANGE_URL }),
   email: schema.maybe(
-    schema.object({
-      domain_allowlist: schema.arrayOf(schema.string()),
-    })
+    schema.object(
+      {
+        domain_allowlist: schema.maybe(schema.arrayOf(schema.string())),
+        recipient_allowlist: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
Can we add this to `src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker` too?

Reply:
thanks, done in [539585c](https://github.com/elastic/kibana/pull/220058/commits/539585c9aca9413a9efb30dc9b1803e497759a37)

Reply:
Actually, that file is NOT auto-generated.  See: https://ela.st/docker-file-not-autogenerated - I keep forgetting to delete that line when I get in that file :-).  Feel free to delete it yourself!

BTW, we will also want to add this to the cloud allow-list of config keys.  We can point you to some example PRs where we've done this - it's fairly simple, but will be another PR since it's a different repo.

Reply:
I removed the message in 9d8c11f94f3ab9f725250f0f55d4ba111696abe1

Feel free to share those PRs here or via slack with me whenever you have time :)